tests/functional/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Do we need this file?

Reply:
Nope, part of cypress template, removed it.